documentation/docs/80-adapter-api.md

```diff
@@ -23,15 +23,17 @@ export default function (options) {
 
 The types for `Adapter` and its parameters are available in [types/config.d.ts](https://github.com/sveltejs/kit/blob/master/packages/kit/types/config.d.ts).
 
-Within the `adapt` method, there are a number of things that an adapter should do:
+Within the `adapt` method, an adapter should
 
 - Clear out the build directory
-- Output code that:
-  - Calls `init`
-  - Converts from the platform's request to a [SvelteKit request](#hooks-handle), calls `render`, and converts from a [SvelteKit response](#hooks-handle) to the platform's
-  - Globally shims `fetch` to work on the target platform. SvelteKit provides a `@sveltejs/kit/install-fetch` helper for platforms that can use `node-fetch`
+- Import `{ appResolver }` from `@sveltejs/kit/adapter`
+- Pass in `appResolver()` to esbuild plugins for build options
```

Comment:
I don't see any other way to include the `../output/server/app.js` other than to have a build step that bundles it, except `adapter-static` of course. I don't know about `adapter-node` though, but it's probably the most flexible of all. Maybe @jthegedus have any thoughts on this?

Reply:
`adapter-node` didn't have bundling for a long time. It was only added in https://github.com/sveltejs/kit/pull/1648. I have mixed feeling about making it more required because I was thinking we should go the other way and make it optional as mentioned in the above comment